keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -77,6 +79,21 @@ File and block are first class citizens within the Kubernetes ecosystem.  Object
 
 # Proposal
 
+## User Stories
+
+#### Admin
+
+- As a cluster administrator, I can set generated buckets' storage capacity via the Kubernete's API, so that workloads may not inordinately contribute to monthly infrastructure costs.
+- As a cluster administrator, I can use Kubernetes RBAC policies around bucket APIs, so that I may control access to existing buckets from within the cluster, reducing the need to adminster an external storage interface.
```

Comment:
This is good. Do we care about policies covering _new_ buckets too?

Reply:
I don't _think_ so, as that would mean prescribing how a user could interact with a bucket they wanted created, which I don't think makes sense.